systemd/openqa-trigger-from-obs-update.service

```diff
@@ -6,6 +6,3 @@ After=network-online.target
 Type=oneshot
 ExecStart=/usr/bin/git -C /opt/openqa-trigger-from-obs pull --quiet --rebase origin master
 User=geekotest
-
-[Install]
-WantedBy=default.target
```

Comment:
This means one can no longer enable this unit normally, e.g. to run the triggering just once after boot. I guess this makes sense for this timer-triggered unit. We also don't have this for other timer-triggered units.